### ISSUE TITLE
Set up proper tests and run on CI

### ISSUE THREAD

**Issue by Datseris** [JuliaDynamics/InteractiveDynamics.jl#70]
At the moment this package is not really tested. Problem is, I don't even know how to test it on CI. I've been advised to copy Makie tests, but I'm not sure what to do!

**Comment by fbanning** [comment]
Have a look at the reference testing done by GraphMakie: https://github.com/MakieOrg/GraphMakie.jl/tree/master/test

Do you think that would be a decent way for us to do it for InteractiveDynamics?

**Comment by Datseris** [comment]
yes this seems legit. I guess we can use the figure returned by any of the plotting functions and do this score testing?
